test/Core.Test/OrganizationFeatures/OrganizationSponsorships/FamiliesForEnterprise/CreateSponsorshipCommandTests.cs

```diff
@@ -35,6 +35,7 @@ private bool SponsorshipValidator(OrganizationSponsorship sponsorship, Organizat
             Organization org, OrganizationUser orgUser, SutProvider<CreateSponsorshipCommand> sutProvider)
         {
             org.PlanType = sponsoringOrgPlan;
+            orgUser.Status = OrganizationUserStatusType.Confirmed;
```

Comment:
Does this fix a flaky test, or is this a change in behavior?

Reply:
Flaky test. I changed the order of the exceptions and it started failing because the user wasn't confirmed